code/modules/client/preferences.dm

```diff
@@ -202,12 +202,12 @@ var/const/MAX_SAVE_SLOTS = 8
 			if(!IsGuestKey(thekey))
 				var/load_pref = load_preferences_sqlite(theckey)
 				if(load_pref)
-					while(!speciesinit)
+					while(!SShumans.initialized)
```

Comment:
`SS_READY(SShumans)`
I don't think it's possible for the controller to be null in this instance, but it's good to be safe